### PR TITLE
feat: Nav bar and footer

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
New styled Navbar and Footer + fontawesome js file for icons
image
   
image

### CODE REVIEW

File: css/footer.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.footer{
+    background-color: var(--color-yellow);
+    text-align: center;
+    color: var(--defaul-color);
```
Comment (shivangisharma01) [inline css/footer.css]: Do check and correct the variable name of color.

**Review by shivangisharma01 — APPROVED** [review]
clean and well structured code, good use of variables.

File: css/footer.css (hunk 1 above)
Comment (Shivanshsharma01) [reply css/footer.css]: sure

File: css/footer.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.footer{
+    background-color: var(--color-yellow);
+    text-align: center;
+    color: var(--default-color);
+    padding: 2rem 1rem;
+    font-weight: 800;
```
Comment (AnuragSahu11) [inline css/footer.css]: you can create variables for font weight too ex, bold regular semibold

**Review by AnuragSahu11 — APPROVED** [review]
Code looks good , just a few suggestions, if you are not using code formater then start using it , because code seems unformated in some places. Also the shade of yellow seems too bright.

File: css/footer.css (hunk 2 above)
Comment (Shivanshsharma01) [reply css/footer.css]: sure

**Comment by Shivanshsharma01** [comment]
Done with suggested changes now merging

**Pull request merged**